source/docs/prismic/expose-content.md

```diff
@@ -0,0 +1,368 @@
+```
+
+The complete list of available Transformers can be found under [the `transformers` directory in Prismic module](https://gitlab.com/front-commerce/front-commerce-prismic/-/tree/0.5.0/prismic/server/modules/prismic/core/loaders/transformers). We will use some of them in the following examples.
+
+## Implementation examples
+
+Those examples assume that you have created [a GraphQL module](https://developers.front-commerce.com/docs/essentials/extend-the-graphql-schema.html#Create-a-new-GraphQL-module) and that it is already registered. Before being able to use the Prismic module API, you have to make sure `Prismic/Core` is a dependency of your module. In addition, we will use the Wysiwyg feature provided by `Front-Commerce/Wysiwyg`, so this one also needs to be in the dependencies:
+
+```diff
+import typeDefs from "./schema.gql";
+import resolvers from "./resolvers";
+
+export default {
+  namespace: "MyModule",
+-  dependencies: [],
++  dependencies: ["Prismic/Core", "Front-Commerce/Wysiwyg"],
+  typeDefs,
+  resolvers,
+}
+```
+
+### Expose a Homepage in the Graph
+
+#### Define the schema
+
+Let's assume you have created a Single Custom Type _Homepage_ which identifier is `homepage`. This Custom Type has three fields:
+
+* `title` of type Title
+* `image` of type Image
+* `text` of type Rich Text
+
+While not mandatory, the best way to expose the Homepage in the Graph is to model the GraphQL type after the Custom Type. So in `schema.gql`, you can add:
+
+```graphql
+type Homepage {
+  title: String
+  image: String
+  text: DefaultWysiwyg
+}
+
+extend type Query {
+  homepage: Homepage
+}
+```
+
+#### Implement the resolver
+
+Once you have defined the type in the Graph, you need to implement the corresponding resolver in `resolvers.js` to retrieve the homepage Content from Prismic:
+
+```js
+export default {
+  Query: {
+    homepage: async (root, args, { loaders }) => {
+      const { TitleTransformer, RichtextToWysiwygTransformer, ImageTransformer } = loaders.Prismic.transformers;
+      const homepage = await loaders.Prismic.loadSingle("homepage", {
+        fieldTransformers: {
+          title: new TitleTransformer(),
+          image: new ImageTransformer(),
+          text: new RichtextToWysiwygTransformer(loaders.Wysiwyg),
+        },
+      });
+      return homepage;
+    },
+  },
+};
+```
+
+#### Image field handling
+
+In this example, without a dedicated resolver, the `image` field of the `Homepage` type is the path to the image of [the `main` view](https://prismic.io/docs/technologies/templating-image-field-javascript#get-an-image-view). In Prismic, while configuring an Image field in the Custom Type editor, it is possible to define several views to get the same image under a different format. Those views can also be exposed in the Graph. For instance, if you have defined a `thumbnail` view, it can be exposed in the graph by applying the following changes. In `schema.gql`:
+
+```diff
+type Homepage {
+  image: String
```

Comment:
For consistency with the first example and the next resolver diff
```suggestion
  title: String
  image: String
```

Reply:
fixed in 7b3b685